docs/sources/write/markdown-guide/index.md

```diff
@@ -242,8 +242,8 @@ The title text does.
 
 ### Examples
 
-- `![Grafana logo](/link/to/grafanalogo/logo.png "Grafana logo")`
-- `![Example](/static/img/docs/folder_name/alert_test_rule.png "Example title")`
+- `![The Grafana logo](/link/to/grafanalogo/logo.png "Grafana logo")`
+- `![An alert test rule](/static/img/docs/folder_name/alert_test_rule.png "Example title")`
```

Comment:
What about using the figure shortcode? Which is preferable? markdown or the shortcode?

```
{{< figure alt="<ALT TEXT>"  src="/media/<PATH TO IMAGE>" >}}
```

Reply:
Mm, that's an interesting question. I think generally it's not important but you will need to use `figure` if you want captions or additional styling.

Reply:
I think that's explained in the next paragraph

Reply:
I clicked expand up.... instead of down. Using the shortcode was/is on my mind because I totally missed that it was available and assumed the markdown was the only way to add images.